apps-devstg/us-east-1/k8s-eks/network/network.auto.tfvars

```diff
@@ -1 +1,2 @@
 vpc_enable_nat_gateway = false
+enable_vpc_flow_logs = false
```

Comment:
This looks redundant since the variable's default value is already set to false.